waspc/data/Generator/templates/server/src/actions/_action.ts

```diff
@@ -7,7 +7,7 @@ import prisma from '../dbClient.js'
           consider in the future if it is worth removing this duplication. =}
 
 export default async function (args, context) {
-  return {= jsFn.importIdentifier =}(args, {
+  return {= jsFn.importIdentifier =}(args as never, {
```

Comment:
This is pretty ridiculous, but is the quickest possible way to fix a type issue caused by the requirements described here: https://github.com/wasp-lang/wasp/pull/1090/files#r1159732471.

We'll come up with a more long-term solution eventually.